fastlane/Fastfile

```diff
@@ -149,74 +174,36 @@ lane :publish_release do |options|
   end
 
   # Write the new changes
-  File.open("../StreamSPM/Package.swift", "w") { |file| file << file_data }
+  File.open("./Package.swift", "w") { |file| file << file_data }
 
   # Update the repo
-  sh("cd ../StreamSPM && git add -A")
+  sh("git add -A")
   sh("git commit -m 'Bump #{version}'")
   sh("git tag #{version}")
   sh("git push origin main --force --tags")
 
   # Clean Up
-  sh("cd ../")
-  sh("rm -r StreamSPM")
-
-  # Create Github Release
-  github_release = set_github_release(
-                     repository_name: "GetStream/stream-chat-swift",
-                     api_token: ENV["GITHUB_TOKEN"],
-                     name: version,
-                     tag_name: version,
-                     description: changes,
-                     upload_assets: ["Products/StreamChat.zip", "Products/StreamChatUI.zip", "Products/StreamChat-All.zip"]
-                   )
-  push_pods(sync: options[:sync])
-
-  # Set tag
-  sh("git tag #{version}")
-
-  # Push tag to remote
-  push_to_git_remote(tags: true)
-
-  UI.success("Github release was created, please visit #{github_release["url"]} to see it")
+  sh("rm -r ../../StreamSPM")
 end
 
 desc "Pushes the StreamChat and StreamChatUI SDK podspecs to Cocoapods trunk"
 lane :push_pods do |options|
+  # First pod release will not have any problems
+  pod_push(path: "StreamChat.podspec", allow_warnings: true)
 
-  def release_pod(sync:, podspec:, dry:)
+  def release_ui(sync)
```

Comment:
This might need to be removed, removes @bielikb 's changes

Reply:
@adamrushy  please revert these changes/update against the latest release candidate ;)

Reply:
@bielikb @polqf done now 👍🏼